services/visual-studio-app-center/visual-studio-app-center-releases-osversion.service.js

```diff
@@ -0,0 +1,60 @@
+'use strict'
+
+const Joi = require('@hapi/joi')
+const {
+  BaseVisualStudioAppCenterService,
+  keywords,
+  documentation,
+} = require('./visual-studio-app-center-base')
+
+const schema = Joi.object({
+  app_os: Joi.string().required(),
+  min_os: Joi.string().required(),
+}).required()
+
+module.exports = class VisualStudioAppCenterReleasesOSVersion extends BaseVisualStudioAppCenterService {
+  static get category() {
+    return 'version'
+  }
+
+  static get route() {
+    return {
+      base: 'visual-studio-app-center/releases/osver',
+      pattern: ':owner/:app/:token',
```

Comment:
Again you'll have to pardon my lack of familiarity with this service 😄 but is there just a single min_os version for an app, or is there one for iOS and one for Android? If so, how would users make that distinction, if they wanted a badge for each?

Reply:
There is a single `min_os` version for each app. Each platform / OS is a separate appcenter project.

Reply:
> Each platform / OS is a separate appcenter project.

I assume that `project` and `app` are synonymous? For example if I had my app for iOS and Android, I'd have badges like

visual-studio-app-center/releases/osver/me/my-android-app/mytoken
visual-studio-app-center/releases/osver/me/my-ios-app/mytoken

Reply:
Yep, sorry for not making it clearer before.

Reply:
No worries, thanks for explaining